src/bootstrap/util.rs

```diff
@@ -301,3 +301,108 @@ pub fn is_valid_test_suite_arg<'a, P: AsRef<Path>>(
         _ => None,
     }
 }
+
+/// Copied from `std::path::absolute` until it stabilizes.
+///
+/// FIXME: this shouldn't exist.
+pub(crate) fn absolute(path: &Path) -> PathBuf {
+    if path.as_os_str().is_empty() {
+        panic!("can't make empty path absolute");
+    }
+    #[cfg(unix)]
+    { t!(absolute_unix(path), format!("could not make path absolute: {}", path.display())) }
+    #[cfg(windows)]
+    { t!(absolute_windows(path), format!("could not make path absolute: {}", path.display())) }
+    #[cfg(not(any(unix, windows)))]
+    {
+        println!("warning: bootstrap is not supported on non-unix platforms");
+        path.to_owned()
```

Comment:
Maybe use `t!(std::env::current_dir()).join(path)` here?

Reply:
Sounds good, but I don't know if it matters in practice - are there any non-unix platforms that have a standard library available?

Reply:
I don't think so.